apps/lib/s_cb.c

```diff
@@ -570,7 +570,9 @@ void msg_cb(int write_p, int version, int content_type, const void *buf,
 {
     BIO *bio = arg;
     const char *str_write_p = write_p ? ">>>" : "<<<";
-    const char *str_version = lookup(version, ssl_versions, "???");
+    char tmpbuf[128];
+    BIO_snprintf(tmpbuf, sizeof(tmpbuf)-1, "Not TLS data or unknown version (version=%d, content_type=%d).", version, content_type);
```

Comment:
This shouldn't be in the variable declarations, move it down please.

Reply:
Done.